packages/components/psammead-brand/src/index.jsx

```diff
@@ -212,6 +215,7 @@ Brand.propTypes = {
   serviceLocalisedName: string,
   borderTop: bool,
   borderBottom: bool,
+  location: string,
```

Comment:
Should this not be `oneOf(['header', 'footer'])` ?

Reply:
I left it open to other possible values considering that location could be anywhere on the page.

Reply:
Okay, makes sense